### DISCUSSION TITLE
Weighted order for tablist

### DISCUSSION THREAD

**Discussion by neokyle** [Zrips/CMI#6907] [Enhancement]
Would list players by groups, so everyone in X group is listed first no the tab list, then everyone in group Y is listed next and so on. Could easily me accomplished with a permission node.

**Comment by CloudeLecaw** [comment]
this is requested a lot of times allready in disdussion may u enjoy one of the discussion there
https://github.com/Zrips/CMI/discussions/5376

https://github.com/Zrips/CMI/discussions/931
https://github.com/Zrips/CMI/discussions/5233
https://github.com/Zrips/CMI/discussions/3571
https://github.com/Zrips/CMI/discussions/3912
https://github.com/Zrips/CMI/discussions/6031
https://github.com/Zrips/CMI/discussions/4695
https://github.com/Zrips/CMI/discussions/1480
https://github.com/Zrips/CMI/discussions/653
https://github.com/Zrips/CMI/discussions/3479
https://github.com/Zrips/CMI/discussions/2671
https://github.com/Zrips/CMI/discussions/1492
